app/src/main/res/values/strings.xml

```diff
@@ -1162,15 +1162,48 @@
     <string name="talk_share_talk_page">Wikipedia - discussions on talk page</string>
     <string name="talk_edit_disclaimer"><![CDATA[This is a talk page. Please respect the talk page guidelines, and remember to sign your posts by typing four tildes: ~~~~ when posting using wikitext.<br\><br\><a href="https://en.wikipedia.org/wiki/Wikipedia:Talk_page_guidelines">Wikipedia:Talk page guidelines</a>]]></string>
     <string name="talk_search_hint">Search messages</string>
+    <string name="talk_search_topics_hint">Search topics</string>
     <string name="talk_overflow_sort_by">Sort by</string>
     <string name="talk_overflow_sort_date_published">Date published</string>
+    <string name="talk_overflow_sort_date_updated">Date updated</string>
     <string name="talk_overflow_sort_topic_name">Topic name</string>
+    <plurals name="talk_show_replies_count">
+        <item quantity="one">Show reply</item>
+        <item quantity="other">Show replies (%d)</item>
+    </plurals>
+    <plurals name="talk_hide_replies_count">
+        <item quantity="one">Hide reply</item>
+        <item quantity="other">Hide replies (%d)</item>
+    </plurals>
+    <string name="talk_thread_subscribed_to">Subscribed to \"%s\"</string>
+    <string name="talk_thread_unsubscribed_from">Unsubscribed from \"%s\"</string>
+    <string name="talk_tab_discussions">Discussions</string>
+    <string name="talk_tab_archived">Archived</string>
+    <string name="talk_list_item_overflow_subscribe">Subscribe</string>
+    <string name="talk_list_item_overflow_subscribed">Subscribed</string>
+    <string name="talk_list_item_overflow_mark_as_read">Mark as read</string>
+    <string name="talk_list_item_overflow_mark_as_unread">Mark as unread</string>
+    <string name="talk_list_item_overflow_share">Share</string>
+    <string name="talk_list_item_last_comment_date">Last comment: %s</string>
+    <string name="talk_list_item_reply_icon_content_description">Reply number</string>
+    <string name="talk_list_item_user_icon_content_description">User icon</string>
+    <string name="talk_list_item_swipe_mark_as_read">Read</string>
+    <string name="talk_list_item_swipe_mark_as_unread">Unread</string>
+    <string name="talk_footer_about_this_talk_page">About this talk page</string>
+    <string name="talk_footer_view_edit_history">View edit history</string>
+    <string name="talk_footer_view_user_profile">View user profile</string>
```

Comment:
"User profile" is not correct Wiki terminology. We have user pages, not user profiles.

Reply:
Yes!